src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -42,21 +51,35 @@
     @GET
     @Path("/")
     @Produces(MediaType.APPLICATION_JSON)
-    public Collection<KabaneroInstance> getAllInstances() {
-        return KabaneroManager.getKabaneroManagerInstance().getAllKabaneroInstances();
+    public KabaneroList getAllInstances() throws IOException, ApiException, GeneralSecurityException {
```

Comment:
we should keep most logic for these endpoints inside kabaneroClient instead of mixed in this file and kabanero client

Reply:
as in just keep this as 1 line that calls `KabaneroClient.getInstances()`